main.cpp

```diff
@@ -89,9 +88,11 @@ struct Point
         return *this;
     }
 
-    Point& multiply(FloatType& f);
-    Point& multiply(DoubleType& d);
-    Point& multiply(IntType& i);
+    Point& operator*=(float m) { return multiply(m); }
```

Comment:
Instructions did not say to add this member function.
You're also not using it in main().
- [ ] remove `operator*=`